tests/integration_tests/README.md

```diff
@@ -0,0 +1,43 @@
+# RPC Integration tests
+
+The RPC integration test suite uses nix and python to send identical queries to both an Ethermint and a [Geth](https://github.com/ethereum/go-ethereum) client and compare the responses. It allows for quickly assessing the compatibility with Geth.
```

Comment:
Thank you for adding this Readme! 🙏 We were lacking this instructions. I would just suggest changing this description as the integration tests are not only concern about Geth but basically any configuration. Maybe something like this?

```
The RPC integration test suite uses nix for reproducible and configurable
builds allowing to run integration tests using python web3 library against 
different Ethermint and Geth clients with multiple configurations.
```

Reply:
Good point, thank you!